packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -204,7 +217,7 @@ export class ContextualMenuBase extends BaseComponent<IContextualMenuProps, ICon
       calloutProps,
       onRenderSubMenu = this._onRenderSubMenu,
       focusZoneProps,
-      getMenuClassNames
+      getMenuClassNames = getContextualMenuClassNames
```

Comment:
This seems like a regression -- Its marked as the default classNames function for `getMenuClassNames`, but was no longer being set.